Test/lit.site.cfg

```diff
@@ -178,7 +174,8 @@ config.substitutions.append( ('%refmanexamples', os.path.join(repositoryRoot, 'd
 config.substitutions.append( ('%os', os.name) )
 config.substitutions.append( ('%ver', ver ) )
 config.substitutions.append( ('%sed', 'sed -E' ) )
-config.substitutions.append( ('%exit', "python " + os.path.join(repositoryRoot, 'Scripts/test-exit.py') ) )
+config.substitutions.append( ('%exits-with', "python " + os.path.join(repositoryRoot, 'Scripts/test-exit.py') ) )
+config.substitutions.append( ('!', "python " + os.path.join(repositoryRoot, 'Scripts/test-exit.py') + " -z" ) )
```

Comment:
Interesting substitution!  How did it work before?

Reply:
the ! command did not work before on original lit (just in the Unit version)

Reply:
Weird, ! was working for me on the original lit at least (e.g. `// RUN: ! false` passed). It's possible it wasn't portable enough to work on all OS' for some reason though. This is a reasonable implementation as well.

Reply:
It did not work for me (so I fixed it) -- note that %runDafnyForEachTest does not (for me) either. I'd like to investigate that, but it is not needed for this PR